sdk/attestation/Azure.Security.Attestation/src/Models/AttestationResult.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Security.Cryptography.X509Certificates;
+using Azure.Core;
+using System.Text.Json.Serialization;
+
+namespace Azure.Security.Attestation.Models
+{
+    [JsonConverter(typeof(AttestationResultConverter))]
+    public partial class AttestationResult
+    {
+
+        internal AttestationResult()
+        {
+        }
+
+        /// <summary>
+        /// Returns the time when this attestation token was issued.
```

Comment:
We generally use "Gets" for getter only property descriptions, i.e. "Gets the time..."
For get/set, we would say "Gets or sets..."

Reply:
Updated. FWIW, there's a stylecop rule which enforces this, it might make sense to add that here.

Reply:
@tg-msft what do you think? Should we look into enabling this stylecop rule?